### PR TITLE
Revert 3.0.0 builds to jdk-17

### PR DESCRIPTION
**Pull request by gaiksaya**
### Description
Reverting to jdk-17 until core finalizes and merges the changes for next jdk version for 3.0.0
cc: @reta 

### Issues Resolved
_List any issues this PR will resolve, e.g. Closes [...]._ 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: manifests/3.0.0/opensearch-3.0.0.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ build:
 ci:
   image:
     name: opensearchstaging/ci-runner:ci-runner-centos7-opensearch-build-v2
-    args: -e JAVA_HOME=/opt/java/openjdk-20
+    args: -e JAVA_HOME=/opt/java/openjdk-17
```
Comment (reta) [inline manifests/3.0.0/opensearch-3.0.0.yml:9]:
```suggestion
    args: -e JAVA_HOME=/opt/java/openjdk-19
```

**Comment by reta** [comment]
Sorry @gaiksaya , I missed this one and opened the duplicate https://github.com/opensearch-project/opensearch-build/pull/3545 (please feel free to close)

**Review by prudhvigodithi — APPROVED** [review]

**Review by peterzhuamazon — APPROVED** [review]

File: manifests/3.0.0/opensearch-3.0.0.yml (hunk 1 above)
Comment (gaiksaya) [reply manifests/3.0.0/opensearch-3.0.0.yml:9]: Yeah went through old PR and just reverted what it was before

Comment (reta) [reply manifests/3.0.0/opensearch-3.0.0.yml:9]: Thank you

**Pull request merged**